.github/workflows/github-action-pypi.yaml

```diff
@@ -8,13 +8,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.11"]
+        python-version: ["3.10"]
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: install gdal lib
```

Comment:
Also, is GDLA needed for building wheels?

Reply:
Yes. It seems like that.